test/integration/coscheduling_test.go

```diff
@@ -34,16 +44,70 @@ import (
 	testutils "k8s.io/kubernetes/test/integration/util"
 	imageutils "k8s.io/kubernetes/test/utils/image"
 
+	scheconfig "sigs.k8s.io/scheduler-plugins/pkg/apis/config"
+	"sigs.k8s.io/scheduler-plugins/pkg/apis/scheduling"
+	"sigs.k8s.io/scheduler-plugins/pkg/apis/scheduling/v1alpha1"
 	"sigs.k8s.io/scheduler-plugins/pkg/coscheduling"
+	pgclientset "sigs.k8s.io/scheduler-plugins/pkg/generated/clientset/versioned"
+	coschedulingutil "sigs.k8s.io/scheduler-plugins/pkg/util"
 	"sigs.k8s.io/scheduler-plugins/test/util"
-	// Ensure scheme package is initialized.
-	_ "sigs.k8s.io/scheduler-plugins/pkg/apis/config/scheme"
 )
 
-var lowPriority, midPriority, highPriority = int32(0), int32(100), int32(1000)
-
 func TestCoschedulingPlugin(t *testing.T) {
+	todo := context.TODO()
+	ctx, cancelFunc := context.WithCancel(todo)
+	testCtx := &testutils.TestContext{
+		Ctx:      ctx,
+		CancelFn: cancelFunc,
+		CloseFn:  func() {},
+	}
 	registry := fwkruntime.Registry{coscheduling.Name: coscheduling.New}
+	t.Log("create apiserver")
+	_, config := util.StartApi(t, todo.Done())
+
+	config.ContentType = "application/json"
+
+	apiExtensionClient, err := apiextensionsclient.NewForConfig(config)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	kubeConfigPath := ""
```

Comment:
Can we package this logic as a public function？ This is easy to used by other  plugin

Reply:
done